webapp/src/main/java/com/box/l10n/mojito/react/ReactAppController.java

```diff
@@ -103,6 +110,13 @@ ReactUser getReactUser() {
               reactUser.setGivenName(currentAuditor.getGivenName());
               reactUser.setSurname(currentAuditor.getSurname());
               reactUser.setCommonName(currentAuditor.getCommonName());
+
+              Role role = Role.USER;
+              Authority authority = authorityRepository.findByUser(currentAuditor);
+              if (authority != null) {
+                role = Role.valueOf(authority.getAuthority().replace("ROLE_", ""));
+              }
```

Comment:
introduce a function?

What's the advantage to removing the "ROLE_" prefix?

Reply:
I have refactored it to use the existing methods in the `UserService`. Additionally, I went ahead with the `ROLE_` enum refactoring to simplify the logic.